packages/api/src/feature_flag/service/create.ts

```diff
@@ -0,0 +1,41 @@
+import { logger } from "@stela/logger";
+import createError from "http-errors";
+import { TinyPgError } from "tinypg";
+import type { CreateFeatureFlagRequest, FeatureFlagRow } from "../models";
+import { db } from "../../database";
+
+const duplicateFeatureFlagError =
+  'duplicate key value violates unique constraint "feature_flag_unique"';
```

Comment:
It looks like the constraint is called `feature_flag_name_unique`

Reply:
I changed it in the migration, but forgot to change it here. Nice catch, thanks :)